src/crystal_lib/type_mapper.cr

```diff
@@ -127,10 +127,34 @@ class CrystalLib::TypeMapper
 
   def map_internal(type : CrystalLib::Enum)
     enum_name = crystal_type_name(check_anonymous_name(type.name))
+
+    # try to find out the type of the enum
+    # (see https://stackoverflow.com/a/1113869)
+    if type.values.any?{|v| v.value < Int32::MIN || v.value > Int32::MAX }
```

Comment:
Minor, but you can flip this and use that sweet syntax :smile:

`if types.values.all? { |v| Int32::MIN <= v.value <= Int32::MAX }`

Reply:
Yes, no problem :)